README.md

```diff
@@ -223,6 +233,176 @@ new ConflictProblem('Duplicated key for book with ID 20.');
 }
 ````
 
+#### NotModifiedProblem
+
+```php
+use Phpro\ApiProblem\Http\NotModifiedProblem;
+
+new NotModifiedProblem('Nothing has changed!.');
+```
+
+```json
+{
+    "status": 304,
+    "type": "http:\/\/www.w3.org\/Protocols\/rfc2616\/rfc2616-sec10.html",
+    "title": "Not Modified",
+    "detail": "Nothing has changed!"
+}
+````
+
+#### MethodNotAllowedProblem
+
+```php
+use Phpro\ApiProblem\Http\MethodNotAllowedProblem;
+
+new MethodNotAllowedProblem('Only POST and GET allowed.');
+```
+
+```json
+{
+    "status": 405,
+    "type": "http:\/\/www.w3.org\/Protocols\/rfc2616\/rfc2616-sec10.html",
+    "title": "Method Not Allowed",
+    "detail": "Only POST and GET allowed."
+}
+````
+
+#### PreconditionFailedProblem
+
+```php
+use Phpro\ApiProblem\Http\PreconditionFailedProblem;
+
+new PreconditionFailedProblem('Incorrect entity tag provided.');
+```
+
+```json
+{
+    "status": 412,
+    "type": "http:\/\/www.w3.org\/Protocols\/rfc2616\/rfc2616-sec10.html",
+    "title": "Precondition Failed",
+    "detail": "Incorrect entity tag provided."
+}
+````
+
+#### UnsupportedMediaTypeProblem
+
+```php
+use Phpro\ApiProblem\Http\UnsupportedMediaTypeProblem;
+
+new UnsupportedMediaTypeProblem('Please provide valid JSON.');
+```
+
+```json
+{
+    "status": 415,
+    "type": "http:\/\/www.w3.org\/Protocols\/rfc2616\/rfc2616-sec10.html",
+    "title": "Unsupported Media Type",
+    "detail": "Please provide valid JSON."
+}
+````
+
+#### IAmATeapotProblem
+
+```php
+use Phpro\ApiProblem\Http\IAmATeapotProblem;
+
+new IAmATeapotProblem('More tea please.');
+```
+
+```json
+{
+    "status": 418,
+    "type": "http:\/\/www.w3.org\/Protocols\/rfc2616\/rfc2616-sec10.html",
+    "title": "I'm a teapot",
+    "detail": "More tea please."
+}
+````
+
+#### UnprocessableEntityProblem
+
+```php
+use Phpro\ApiProblem\Http\UnprocessableEntityProblem;
+
+new UnprocessableEntityProblem('Unable to process the contained instructions.');
+```
+
+```json
+{
+    "status": 422,
+    "type": "http:\/\/www.w3.org\/Protocols\/rfc2616\/rfc2616-sec10.html",
+    "title": "Unprocessable Entity",
+    "detail": "Unable to process the contained instructions."
+}
+````
+
+#### LockedProblem
+
+```php
+use Phpro\ApiProblem\Http\LockedProblem;
+
+new LockedProblem('This door is locked.');
+```
+
+```json
+{
+    "status": 423,
+    "type": "http:\/\/www.w3.org\/Protocols\/rfc2616\/rfc2616-sec10.html",
+    "title": "Locked",
+    "detail": "This door is locked."
+}
+````
+
+#### FailedDependencyProblem
```

Comment:
>    The 424 (Failed Dependency) status code means that the method could
   not be performed on the resource because the requested action
   depended on another action and that action failed.  For example, if a
   command in a PROPPATCH method fails, then, at minimum, the rest of
   the commands will also fail with 424 (Failed Dependency).

Can we add the dependency to the problem?